frontend/src/container/NewWidget/index.tsx

```diff
@@ -423,35 +424,38 @@ Panel type: ${graphType}
 Dashboard Id: ${selectedDashboard?.uuid || ''}
 				
 Thanks`}
-				/>
-				<ButtonContainer>
-					{isSaveDisabled && (
-						<Tooltip title={MESSAGE.PANEL}>
+					/>
+				</Col>
+				<Col span={8} offset={8}>
```

Comment:
use css classes and remove this `antd` components

Reply:
removed and used div with classname